test/general/TestFill.py

```diff
@@ -430,34 +430,6 @@ def test_excess_priority_distribute(self):
 
         self.assertFalse(locations[3].item.advancement)
 
-    def test_multiple_world_distribute(self):
```

Comment:
I removed this test because I don't think it adds a lot of value and it is very fragile to changes in the fill logic.